core/statistics/machine/memStatistics.go

```diff
@@ -21,8 +22,23 @@ func (ms *MemStatistics) ComputeStatistics() {
 		return
 	}
 
+	currentProcess, err := getCurrentProcess()
+	if err != nil {
+		ms.setZeroStatsAndWait()
+		return
+	}
+
+	processMemoryUsage, err := currentProcess.MemoryInfo()
```

Comment:
err not checked

Reply:
err is checked now. If err is not nil set all memory statistics with 0.